.werft/build.js

```diff
@@ -241,7 +252,7 @@ async function issueAndInstallCertficate(namespace, domain) {
             break;
         }
 
-        sleep(5000);
+        sleep(3000);
```

Comment:
I like that!

Reply:
(Note: https is not used atm. We'll have to investigate why certmanager works only sometimes)